source/vscode/src/test/extension.test.ts

```diff
@@ -0,0 +1,18 @@
+//
+// Note: This example test is leveraging the Mocha test framework.
+// Please refer to their documentation on https://mochajs.org/ for help.
+//
+
+// The module 'assert' provides assertion methods from node
+import * as assert from 'assert';
+
+
+// Defines a Mocha test suite to group tests of similar kind together
+suite("Extension Tests", () => {
+
+	// Defines a Mocha unit test
+	test("Something 1", () => {
+		assert.equal(-1, [1, 2, 3].indexOf(5));
+		assert.equal(-1, [1, 2, 3].indexOf(0));
+	});
+});
```

Comment:
is this scaffolding for future tests?

Reply:
yah, this was autogenerated by the template generator for plugins.  I have no idea how to actually unit test a vscode plugin 

---
In reply to: [315314829](https://github.com/microsoft/AdaptiveCards/pull/3263#discussion_r315314829) [](ancestors = 315314829)